tests/v2_validation/cattlevalidationtest/core/test_ldap_auth.py

```diff
@@ -74,6 +74,11 @@ def get_authed_token(username=None, password=None):
     return token
 
 
+def delete_ldap_token(id, cookies):
+    response = requests.delete(cattle_url() + '/token/' + id, cookies=cookies)
```

Comment:
Enhance this test to assert that the set-cookie header is there and it is set properly